clusterDeployer.py

```diff
@@ -47,7 +51,7 @@ def setup_dhcp_entry(h: host.Host, cfg: NodeConfig) -> None:
     ret = h.run_or_die(cmd)
     if f"'{name}'" in ret.out:
         logger.info(f"{name} already configured as static DHCP entry - removing before adding back with proper configuration")
-        host_xml = f"<host name='{name}'/>"
+        host_xml = f"<host naversion_cluster_config"
```

Comment:
This is not correct XML. @vrindle  I am wondering why your previous run worked.

Reply:
I am not sure. I know it definitely runs properly because I tested my changes out before asking for another round of reviews. In any case I will fix this.